src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5558,6 +5572,19 @@ protected override void VisitCatchBlock(BoundCatchBlock node, ref LocalState fin
             base.VisitCatchBlock(node, ref finallyState);
         }
 
+        public override BoundNode VisitAttribute(BoundAttribute node)
+        {
+            VisitArguments(node, node.ConstructorArguments, ImmutableArray<RefKind>.Empty, node.Constructor, argsToParamsOpt: node.ConstructorArgumentsToParamsOpt, expanded: node.ConstructorExpanded);
+            foreach (var assignment in node.NamedArguments)
+            {
+                Visit(assignment);
+            }
+
+            var result = base.VisitAttribute(node);
```

Comment:
>var result = base.VisitAttribute(node); [](start = 12, length = 39)

This can be skipped since there we are not relying on any behavior in the base implementation.